support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -753,11 +754,10 @@ export function CheckoutComponent({
 							promotion,
 						)}
 						headerButton={
-							productKey === 'GuardianLight' ? (
-								<BackButton path={`/guardian-light`} buttonText="Back" />
-							) : (
-								<BackButton path={`/${geoId}/contribute`} buttonText="Change" />
-							)
+							<BackButton
+								path={`javascript:history.back()`}
```

Comment:
If someone had a direct link to the checkout (they'd bookmarked it or something) would we want to rely on the browser back functionality here rather than controlling exactly where they go?

Reply:
Ok, changing the logic can assess later, have reset to always return the landing pages (as it was).